charts/kube-state/values.yaml

```diff
@@ -43,6 +43,9 @@ extraArgs:
 
 priorityClassName: ~
 
+# works only with namespace pools
+extraNamespaceList: []
```

Comment:
let's keep this under namespace pools feature flag itself?
https://github.com/astronomer/astronomer/blob/20b78234218db8a06c741c655d6425873bb16606/values.yaml#L42

Reply:
agree but if this is contextual only to namespace pools, we might as well nest the flag for ease of discoverability